### PR TITLE
T2 datadog 2021 03 02

### PR DESCRIPTION
**Pull request by zhangyan133**
https://github.com/Azure/sdk-release-request/issues/1314
--

### CODE REVIEW

File: sdk/datadog/azure-mgmt-datadog/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Release History
 
+## 2.0.0 (2021-03-02)
```
Comment (00Kai0) [inline sdk/datadog/azure-mgmt-datadog/CHANGELOG.md:3]: change to: `1.0.0b3`

File: sdk/datadog/azure-mgmt-datadog/README.md [hunk 2]
```diff
@@ -23,3 +23,13 @@ section of the project.
 
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fazure-mgmt-datadog%2FREADME.png)
+# Usage
```
Comment (00Kai0) [inline sdk/datadog/azure-mgmt-datadog/README.md:26]: Delete this  usage.

File: sdk/datadog/azure-mgmt-datadog/azure/mgmt/datadog/_version.py [hunk 3]
```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "2.0.0"
```
Comment (00Kai0) [inline sdk/datadog/azure-mgmt-datadog/azure/mgmt/datadog/_version.py:9]: change to `1.0.0b3`

**Review by 00Kai0 — CHANGES_REQUESTED** [review]

**Review by 00Kai0 — APPROVED** [review]

**Review by msyyc — APPROVED** [review]

**Pull request merged**